docs/content/developers/building-contributing.md

```diff
@@ -72,7 +72,7 @@ rm ~/bin/ddev
 
 [Gitpod](https://www.gitpod.io) provides a quick, preconfigured DDEV experience in the browser for testing a PR easily without the need to set up an environment. For any PR you can use the URL `https://gitpod.io/#https://github.com/ddev/ddev/pull/<YOUR-PR>` to open that PR and build it in Gitpod.
 
-It also allows you to work on the DDEV master branch and test modifiying DDEV's source code.
+It also allows you to work on the DDEV main branch and test modifiying DDEV's source code.
```

Comment:
I'm not advocating for reviewing all these instances, but would it be better to put `main` in backquotes? There's also a longstanding typo here, "modifiying". Not sure how the spellcheck didn't catch that?

```suggestion
It also allows you to work on the DDEV `main` branch and test modifying DDEV's source code.
```